.github/actions/codesigner-SSL/action.yml

```diff
@@ -0,0 +1,91 @@
+name: "codesigner"
+description: "GitHub Action for CodeSigner by SSL.com"
+
+inputs:
+  file_path:
+    description: "Path of the code object to be signed. Supported file types: acm, ax, bin, cab, cpl, dll, drv, efi, exe, mui, ocx, scr, sys, tsp, msi, ps1, ps1xml, js, vbs, wsf, jar"
+    required: true
+  output_dir_path:
+    description: "Directory where the signed code object will be written (leave empty if override is set to true)."
+    required: false
+  malware_block:
+    description: "Scans your file for possible malware to prevent code compromise and prevents signing code if malware is detected."
+    required: false
+    default: "false"
+  override:
+    description: "Override the input file after signing if this argument is set to true and no output_dir_path argument was passed."
+    required: false
+    default: "true"
+  clean_logs:
+    description: "Clean log files after code signing operations."
+    required: false
+    default: "false"
+  jvm_max_memory:
+    description: "Maximum JVM heap size."
+    required: false
+    default: "1024M"
+  signing_method:
+    description: "Code signing method. Default is v1. Supported values: v1, v2."
+    required: false
+    default: "v1"

```

Comment:
I could not find the meaning of v1 and v2 in the SSL.com docu.
Keeping it here as an input just in case :)